02-molecules/cards/cards.stories.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import Card from './Card/Card.component';
+// import NewsCard from './news/NewsCard.component';
+
+/**
+ * Storybook Definition.
+ */
+export default { title: 'Molecules/Cards' };
+
+// Component Examples
+export const card = () => (
+  <Card
+    outputImage={true}
+    image={{
+      alt: 'Random Image from Picsum',
+      src: 'https://picsum.photos/400/300.jpg',
```

Comment:
Do you want to change the story picture sizes to match the Twig storybook? https://picsum.photos/1280/720 https://dev-westernuni.pantheonsite.io/storybook/?path=/story/molecules-cards--card

Reply:
Sure. Done